relational/table.py

```diff
@@ -116,24 +168,65 @@ def _gen_facts(self, row, proj):
             row[self.index(self.PERMIT_AUTHORITY)] = PPTS.OUTPUT_NAME
             row[self.index(self.PERMIT_AUTHORITY_ID)] = proj.field(
                 'fk', PPTS.NAME)
+        elif proj.field('project_id', MOHCDPipeline.NAME) != '':
```

Comment:
Not for this change but just realized we don't have PTS here? They've got a similar scheme to mohcd (street number, name, etc) so might want to pull out that logic to a common are once you get around to that

Reply:
Probably worth a TODO to just clcean this up, I think we can commonize the 'address building' logic into a function somewhere so we dont' have to dupe for different sources?